src/build.ts

```diff
@@ -354,4 +373,13 @@ export class FileBuildEffects implements BuildEffects {
     await prepareOutput(destination);
     await writeFile(destination, contents);
   }
+  async writeBuildManifest(buildManifest: BuildManifest): Promise<void> {
+    const destination = join(this.cacheDir, "_buildManifest.json");
```

Comment:
Can we call this `_build.json` or `_manifest.json` instead? I’m not a big fan of intercapped and potentially case-sensitive file names.